fs/fs_zenfs_test.cc

```diff
@@ -0,0 +1,49 @@
+#include "fs_zenfs.h"
+
+#include <gtest/gtest.h>
+
+#include <cstdlib>
+
+#include "rocksdb/convenience.h"
+#include "rocksdb/env.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+class FsZenFSTest : public testing::Test {
+ public:
+  std::shared_ptr<FileSystem> fs_;
+
+  FsZenFSTest() {}
+
+ protected:
+  void SetUp() override {
+    ConfigOptions config_options;
+    config_options.ignore_unsupported_options = false;
+    char* fs_uri_env = std::getenv("DEVICE");
```

Comment:
This can be "zns_device" or just "device"

Reply:
Not sure what you mean. The `DEVICE` environment variable? It should be a zoned storage device. I'm using `null_blk` for testing.

Reply:
Changed the naming of the variables.